src/asmcnc/apps/start_up_sequence/data_consent_app/screens/wifi_and_data_consent_3.py

```diff
@@ -218,7 +218,11 @@ def __init__(self, **kwargs):
 		self.update_strings()
 		self.set_checkbox_default()
 
-		self.scroll_privacy_notice.privacy_notice.source = self.privacy_notice_path + self.l.lang + '.rst'
+		# Can't seem to use non english letters for file source so filename is different
+		if self.l.lang == 'Français (FR)':
+			self.scroll_privacy_notice.privacy_notice.source = self.privacy_notice_path + 'Francais (FR).rst'
+		else:
+			self.scroll_privacy_notice.privacy_notice.source = self.privacy_notice_path + self.l.lang + '.rst'
 
```

Comment:
just realised that this is a duplicate of what's in update_strings() function anyway, so can likely be deleted. again, my fault originally

Reply:
Not sure why I duplicated it, must've not noticed the update_strings function being called initially, will delete